bot/exts/moderation/infraction/infractions.py

```diff
@@ -27,37 +29,45 @@
     from bot.exts.moderation.watchchannels.bigbrother import BigBrother
 
 
+MAXIMUM_TIMEOUT_DAYS = timedelta(days=28)
+
+
 class Infractions(InfractionScheduler, commands.Cog):
     """Apply and pardon infractions on users for moderation purposes."""
 
     category = "Moderation"
     category_description = "Server moderation tools."
 
     def __init__(self, bot: Bot):
-        super().__init__(bot, supported_infractions={"ban", "kick", "mute", "note", "warning", "voice_mute"})
+        super().__init__(bot, supported_infractions={"ban", "kick", "timeout", "note", "warning", "voice_mute"})
 
         self.category = "Moderation"
-        self._muted_role = discord.Object(constants.Roles.muted)
+        self._muted_role = discord.Object(constants.Roles.muted)  # TODO remove when no longer relevant.
```

Comment:
If you approve my suggestion from above to keep the role, then we'd better rename this